### ISSUE TITLE
Tile cover image a11y

### ISSUE THREAD

**Issue by ppvg** [minvws/nl-rdo-manon#419]
From independent accessibility report, SC 1.3.1:

<blockquote lang="nl">Op pagina <a href="https://minvws.github.io/nl-rdo-manon/documentation/components/image-cover.html">components/image-cover.html</a> staan een aantal voorbeelden van een tegelweergave. De afbeelding die bij de betreffende tegel hoort staat in de code steeds boven de kop die bij die tegel hoort. Dit kan tot problemen leiden, omdat gebruikers dan kunnen denken dat deze content hoort bij een andere bovenliggende kop. Dit speelt vooral voor gebruikers die de webpagina niet visueel zien, maar afhankelijk zijn van de volgorde in de code, bijvoorbeeld bij het gebruik van hulpsoftware. Dit kan worden opgelost door in de code de content daadwerkelijk onder de kop te zetten waar het bij hoort. Via CSS kan het dan zo opgemaakt worden dat er visueel niets hoeft te veranderen. Dit kan bijvoorbeeld via de eigenschap "order".</blockquote>

Tiles with image covers should have the image after the heading in the markup.

**Comment by HeleenSG** [comment]
Solved in: https://github.com/minvws/nl-rdo-manon/pull/1272
